test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -113,7 +113,7 @@ Describe "Start-Process" -Tags @("Feature") {
     }
 
     It "Should start notepad.exe with ShellExecute" -Skip:(!$isFullWin) {
-        $process = Start-Process notepad -PassThru -WindowStyle Normal
+        $process = Start-Process notepad.exe -PassThru -WindowStyle Normal
```

Comment:
We should not use `notepad.exe`. We have a `testexe` is path for the tests that we can use. 

https://github.com/PowerShell/PowerShell/blob/53e6ec6eadc2f363fc671e96b34ade4dda3fe8eb/test/powershell/Language/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1#L12

Reply:
This would need a change in `testexe` so it is out of scope for this PR.